### PR TITLE
add: new environment image tag parsing aliases

### LINKED ISSUE #1531: Detailed version parsing for TensorFlow / PyTorch images
### Main idea

I tried to register an image: `<registry>/ngc-tensorflow:22.05-tf2.8-py38`. It worked well, but the text in the "Base Environment" is displayed as "tf2.8".

image

### Alternative ideas

It is okay and there is no problem in using the image, but it might be visually better to parse the minor version of the major ML framework images to display as "TensorFlow 2.8", "PyTorch 1.10", etc.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by inureyes**
This PR resolves #1531

### CODE REVIEW

File: src/components/backend-ai-resource-broker.ts [hunk 1]
```diff
@@ -169,6 +170,8 @@ export default class BackendAiResourceBroker extends BackendAIPage {
             }
           }
         }
+        this.imageTagAlias = json.tagAlias;
+        console.log(this.imageTagAlias);
```
Comment (agatha197) [inline src/components/backend-ai-resource-broker.ts]: Please remove this line :)

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]
Thanks to handle it! It still looks good now, but can I modify it to automatically convert by pattern? (For example, to convert from tf2.* to TensorFlow 2.*)

**Comment by inureyes** [comment]
> Thanks to handle it! It still looks good now, but can I modify it to automatically convert by pattern? (For example, to convert from tf2.* to TensorFlow 2.*)

It will be better for specific patterns such as tf2.*, pytorch1.*, pytorch2.* and cuda*.*

**Comment by Sujin-Kim1** [comment]
> > Thanks to handle it! It still looks good now, but can I modify it to automatically convert by pattern? (For example, to convert from tf2.* to TensorFlow 2.*)
> 
> It will be better for specific patterns such as tf2._, pytorch1._, pytorch2.* and cuda*.*

I added some codes that replace the tags that have specific patterns.

**Review by adrysn — APPROVED** [review]
LGTM.

**Pull request merged**